packages/kbn-test/src/functional_tests/lib/fips_overrides.ts

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+// This will only apply overrides when running in FIPS mode
+export function applyFipsOverrides(vars: any) {
+  if (process.env.FTR_ENABLE_FIPS_AGENT?.toLowerCase() === 'true') {
```

Comment:
_question_ Maybe we can move statement out of this function and check it explicitly before we call `readConfigFile` ? 

It feels like `applyFipsOverrides` is called by default, but its logic fully depends `process.env.FTR_ENABLE_FIPS_AGENT`